ftw/servicenavigation/profiles/default_plone5/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<metadata>
+    <dependencies>
+        <dependency>profile-collective.z3cform.datagridfield:default</dependency>
+        <dependency>profile-plone.formwidget.contenttree:default</dependency>
```

Comment:
We don't use plone.formwidget.contenttree on Plone 5 :-) ftw.referencewidget IIRC